tsconfig.json

```diff
@@ -6,5 +6,8 @@
     "rootDir": "src",
     "alwaysStrict": true,
     "esModuleInterop": true
-  }
+  },
+  "exclude": [
+    "test"
+  ]
```

Comment:
Surprised I have to include this. Maybe `rootDir` doesn't do what I think it does?

Reply:
You may need to re-add `node_modules` here, I'm not entirely sure. I think `rootDir` is just where your source that will be emitted is, the project will be relative to the `tsconfig.json`. `tsc` will still try to type-check things not in `rootDir`.
> The "exclude" property defaults to excluding the node_modules, bower_components, jspm_packages and <outDir> directories when not specified.

Reply:
https://github.com/microsoft/TypeScript/issues/9858 is relevant.

Reply:
I read that as use `include` instead.

Reply:
https://github.com/libero/article-store/pull/8